cygrid/kernels.pyx

```diff
@@ -72,6 +72,23 @@ cdef double gaussian_1d_kernel(
     return exp(-distance * distance * kernel_params[0])
 
 
+#cdef double nearest_kernel(
```

Comment:
Is it necessary to have commented-out code in the commit? If no, please remove to keep the repository clean.